packages/reports/addon/components/navi-email-input.js

```diff
@@ -25,7 +25,9 @@ export default Component.extend({
   /**
    * @property {Array} emails
    */
-  emails: computed(() => []),
+  emails: computed(function() {
+    return [];
+  }),
```

Comment:
```suggestion
  emails: [],
```

Reply:
I think he'd have to make this into a native class to do that

Reply:
Yup, gonna close these since they're not native classes